conjur/controller/init_controller.py

```diff
@@ -89,10 +89,10 @@ def get_server_certificate(self):
         fingerprint, fetched_certificate = self.init_logic.get_certificate(url.hostname, url.port)
 
         sys.stdout.write(f"\nThe Conjur server's certificate SHA-1 fingerprint is:\n{fingerprint}\n")
-        sys.stdout.write("\nTo verify this certificate, it is recommended to run the following "
+        sys.stdout.write("\nTo verify this certificate, we recommend running the following "
                          "command on the Conjur server:\n"
                          "openssl x509 -fingerprint -noout -in ~conjur/etc/ssl/conjur.pem\n\n")
-        trust_certificate = input("Trust this certificate? (Default=no): ").strip()
+        trust_certificate = input("Trust this certificate? yes/no (Default: no): ").strip()
```

Comment:
Please update other parts of the codebase that has this (Default=) to ensure uniformity. For example for all file overwrites `(Overwrite? (Default=yes)`, `Trust this certificate? (Default=no)` etc